### ISSUE TITLE
Gnome theme colours

### ISSUE THREAD

**Issue by dmgeurts** [paradoxxxzero/gnome-shell-system-monitor-applet#599]
Great plugin, thank you.

Have noticed however that the window (on click) can't be styled. I see images where the background is dark but on my machine, it's just very light grey with slightly darker grey text. It would be great if the system color scheme could be followed.

**Comment by matasarei** [comment]
Agree, we need font and background color settings at least. Gray color is almost invisible on white background.
